android/app/src/main/java/com/zulipmobile/MainActivity.kt

```diff
@@ -7,12 +7,31 @@ import android.webkit.WebView
 import com.facebook.react.ReactActivity
 import com.facebook.react.ReactActivityDelegate
 import com.facebook.react.ReactApplication
+import com.facebook.react.ReactInstanceManager
+import com.facebook.react.ReactNativeHost
 import com.facebook.react.ReactRootView
 import com.facebook.react.bridge.ReactContext
+import com.facebook.react.modules.core.DeviceEventManagerModule
 import com.zulipmobile.notifications.*
 import com.zulipmobile.sharing.handleSend
 import expo.modules.ReactActivityDelegateWrapper
 
+// A convenience shortcut.
+fun ReactContext.emit(eventName: String, data: Any?) {
```

Comment:
Sure, makes sense.

Let's give this a more specific name, since we're putting it in a less specific context: like `emitEvent`.